cmake/tribits/win_interface/include/gettimeofday.c

```diff
@@ -1,4 +1,4 @@
-#include < time.h >
+#include <time.h>
```

Comment:
This is fine obviously.  This is mirrored in https://github.com/TriBITSPub/TriBITS/pull/371.  It should be fine to commit this to Trilinos 'develop'.  I will take care of any merge conflict with the next update of TriBITS to Trilinos 'develop'.